DailyKit/CallManager/CallManager.swift

```diff
@@ -49,6 +51,9 @@ public final class CallManager: CallManageable {
 
         // Apply the defaults to be used before joining a call.
         applyDefaults()
+
+        // Setup observers to manage video publishing when entering the background.
```

Comment:
```suggestion
        // Set up observers to manage video publishing when entering the background.
```

(Super nit; "setup" would be the noun, "set up" would be the verb form.)